src/router/user.routes.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import verify from "../middlewares/verifyToken";
+import { UserController } from "../controllers/UserController";
+const router = express.Router();
+
+router.post("/signup", UserController.createUser);
+
+router.post("/login", UserController.loginUser);
+router.delete("/:id", verify, UserController.deleteUser);
```

Comment:
use understandable routes like "/user/:id"

Reply:
it is "/user/:id" currently the reason why you don't see it here it is because I set /user as base router in index.js